.cron.yml

```diff
@@ -0,0 +1,11 @@
+# Definitions for jobs that run periodically.  For details on the format, see
+# `taskcluster/taskgraph/cron/schema.py`.  For documentation, see
+# `taskcluster/docs/cron.rst`.
+---
+
+jobs:
+    - name: l10-screenshots
+      job:
+          type: decision-task
+          target-tasks-method: l10n_screenshots
+      when: []  # Manual trigger only
```

Comment:
I will give the link later after this lands.

Reply:
Here's the link https://firefox-ci-tc.services.mozilla.com/hooks/project-releng/cron-task-mozilla-mobile-firefox-ios%2Fl10-screenshots.

It doesn't work yet (see https://github.com/mozilla-mobile/firefox-ios/pull/6529). I'll send this link out once I know it's working.